### ISSUE TITLE
Set minimum colour for WCAG contrast guidelines

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#449]
There are a few places in the CSS where text colour is set to #999999. Here's an example:

```
/* line 97, /home/blacklight/SearchWorks/shared/bundle/ruby/2.1.0/gems/bootstrap-sass-3.1.1.1/vendor/assets/stylesheets/bootstrap/_type.scss */
.text-muted, .appliedFilter .filterName:after, .facet-values li .remove, .search_history .filterName, .search_history .filterSeparator {
  color: #999999;
}
```

The lightest colour that passes WCAG tests on our putty background FBFBF9 is 737373. In some cases, when we know the text will appear on a white background (e.g. in dropdown menus) then 767676 passes.

So we need to replace the 999999s that we are actually using with either 737373 (default) or 767676 when we know the background is white.
